apps/meteor/server/models/raw/LivechatDepartment.ts

```diff
@@ -83,6 +83,12 @@ export class LivechatDepartmentRaw extends BaseRaw<ILivechatDepartment> implemen
 				},
 				sparse: true,
 			},
+			{
+				key: {
+					archived: 1,
+				},
+				sparse: true,
+			},
```

Comment:
per my previous comment, this index won't be used by the query on `findNotArchived`, so need to have it.
```suggestion
```

Reply:
Oh no, this index is not for _that_ query. Is for the findArchived we have. (That query is outside the model apparently)

Didn't want to do a PR solely for this

Reply:
oh, ok then =)